app/assets/stylesheets/application.scss

```diff
@@ -1,19 +1,32 @@
 @charset "utf-8";
 @import "normalize-rails";
 @import "bootstrap/bootstrap";
-
 body, html {
   height: 100%;
   width: 100%;
-  background-image: -webkit-linear-gradient(#61CCA2 0%, #B29772 100%);
+  background: #89ccbe;
 }
 h1, h2, h3, h4, h5, h6 {
-  font-family: 'bebasregular', 'open sans', 'helvetica', sans-serif;
+  font-family: 'open sans', 'helvetica', sans-serif;
```

Comment:
Properties should be ordered color, font-family, text-transform<br>Each selector in a comma sequence should be on its own single line